extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php

```diff
@@ -111,6 +111,10 @@ public function getMenuNodes() {
 			[3] // max 3 links
 		);
 
+		if ( $wgLang->getCode() !== 'en' ) {
```

Comment:
What do you think about moving this `if` higher and don't fetch links when they aren't required?